### PR TITLE
Portuguese Certification (mpaa)

### PR DESCRIPTION
**Pull request by Rootzpower**
I add icons and option for Portuguese certification (mpaa)
Icons are also updated on wyrm65 repo.
Can you check if it is ok or need to change more xml that escape me?

### CODE REVIEW

**Comment by Rootzpower** [comment]
I'm doing something wrong because don't work :(

**Comment by jurialmunkey** [comment]
Missing `>` on line 103 of Includes_MPAA.xml

File: 1080i/Includes_MPAA.xml [hunk 1]
```diff
@@ -82,8 +82,106 @@
...
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/6) | String.IsEqual(ListItem.MPAA,M/6) | String.IsEqual(ListItem.MPAA,Portugal M6)">Portugal M6.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/12) | String.IsEqual(ListItem.MPAA,M/12) | String.IsEqual(ListItem.MPAA,Portugal M12)">Portugal M12.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/14) | String.IsEqual(ListItem.MPAA,M/14) | String.IsEqual(ListItem.MPAA,Portugal M14)">Portugal M14.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/16) | String.IsEqual(ListItem.MPAA,M/16) | String.IsEqual(ListItem.MPAA,Portugal M16)">Portugal M16.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated M/18) | String.IsEqual(ListItem.MPAA,M/18) | String.IsEqual(ListItem.MPAA,Portugal M18)">Portugal M18.png</value>
+        <value condition="String.IsEqual(ListItem.MPAA,Rated E) | String.IsEqual(ListItem.MPAA,E) | String.IsEqual(ListItem.MPAA,Australia E)">Australia E.png</value
```
Comment (jurialmunkey) [inline 1080i/Includes_MPAA.xml:103]: Missing `>` on `</value`

That's why it isn't working. 

I've got a script that generates the Includes_MPAA file properly based upon filenames, so no need to do this by hand.

**Comment by jurialmunkey** [comment]
Can you test with that change I pushed to your fork: https://github.com/jurialmunkey/skin.arctic.horizon.2/pull/771/commits/29f2b84cf56e0900e2af7dd12418621c0e59dfde
https://github.com/Rootzpower/skin.arctic.horizon.2

Pretty sure it is working correctly now


BTW This is the script I use to generate the MPAA variables:
https://gist.github.com/jurialmunkey/1b6308cd1ccc7e1f603402457f57b2a9

**Comment by Rootzpower** [comment]
Thanks!!!

**Comment by Rootzpower** [comment]
I don't know what happens but still don't show Portuguese certifications icons

screenshot00000
Inkedscreenshot00002_LI
Inkedscreenshot00001_LI

**Comment by jurialmunkey** [comment]
Because your ratings are not Portuguese. PG and R do not exist as Portuguese ratings so the variable falls back to USA as that's what you've scraped your ratings as.

Need to change your scraper settings to get Portuguese classifications.

**Pull request merged**